dynamic_db_router/router.py

```diff
@@ -83,45 +89,33 @@ def lowest_id_account():
     def __init__(self, database, read=True, write=False):
         self.read = read
         self.write = write
+        self.database = database
         self.created_db_config = False
-        if isinstance(database, str):
```

Comment:
@davidbm04 let's restore these other if clauses for opening a PR with the original repo.